public/app/core/controllers/json_editor_ctrl.ts

```diff
@@ -5,7 +5,7 @@ export class JsonEditorCtrl {
   /** @ngInject */
   constructor($scope) {
     $scope.json = angular.toJson($scope.model.object, true);
-    $scope.canUpdate = $scope.model.updateHandler !== void 0 && $scope.contextSrv.isEditor;
+    $scope.canUpdate = $scope.model.updateHandler !== void 0 && ($scope.model.canUpdate || false) === true;
```

Comment:
Probably something typescript-specific I'm missing here, but, wouldn't `$scope.model.canUpdate` be enough? Instead of the whole or false === true, I mean.

Reply:
yes, agree, looks a bit strange

Reply:
Will fix :+1: